eslint.config.mjs

```diff
@@ -0,0 +1,29 @@
+// @ts-check
+
+import eslint from '@eslint/js'
+import tseslint from 'typescript-eslint'
+
+export default tseslint.config({
+    ignores: [
+        '**/*.js',
+        'node_modules/**',
+        'src/abi/**',
+        'src/types/**',
+        'src/model/**',
+        'eslint.config.mjs',
+    ],
+    languageOptions: {
+        parser: tseslint.parser,
+        parserOptions: {
+            projectService: true,
+            ecmaVersion: 'latest',
+            tsconfigRootDir: './tsconfig.json',
+        },
+    },
+    rules: {
```

Comment:
I think it's because the automatically generated files yield a lot of the unused variable errors. If you remove 'src/types/**' from the `ignores` field and run lint again, you'll see.

But now that we ignore those files anyway, I think it's good to enable it again 👍